gateway/assets/error.html

```diff
@@ -0,0 +1,86 @@
+</head>
+<body>
+  <div id="page-header">
+  <div id="page-header-logo" class="ipfs-logo">&nbsp;</div>
+  <div id="page-header-menu">
+    <div class="menu-item-wide"><a href="https://ipfs.tech" target="_blank" rel="noopener noreferrer">About IPFS</a></div>
+    <div class="menu-item-wide"><a href="https://ipfs.tech#install" target="_blank" rel="noopener noreferrer">Install IPFS</a></div>
+    <div class="menu-item-narrow"><a href="https://ipfs.tech" target="_blank" rel="noopener noreferrer">About</a></div>
+    <div class="menu-item-narrow"><a href="https://ipfs.tech#install" target="_blank" rel="noopener noreferrer">Install</a></div>
+    <div>
+      <a href="https://github.com/ipfs/kubo/issues/new/choose" target="_blank" rel="noopener noreferrer" title="Report a bug">
```

Comment:
I don't think creating Kubo issues here makes sense.

Reply:
This is the header that is showed in all pages from the gateway. Not the error page itself. Please take a look at the `assets/src` files.